client/pages/join-game.jsx

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
+import { io } from 'socket.io-client';
 import AddPostButton from '../components/add-post-button';
 import Post from '../components/post';
+import parseRoute from '../lib/parse-route';
 
 export default class JoinGame extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      posts: null
+      posts: null,
+      socket: io()
     };
+    this.loadGames = this.loadGames.bind(this);
   }
 
   componentDidMount() {
+    const { socket } = this.state;
+    socket.on('game joined', () => {
+      this.loadGames();
+    });
```

Comment:
if the server actually sent the game information in the event payload, you wouldn't have to redownload the universe every time somebody joined a game.